frontend/src/components/custom-tools/combined-output/CombinedOutput.jsx

```diff
@@ -156,15 +168,12 @@ function CombinedOutput({ docId, setFilledFields }) {
         url = publicDefaultOutputApi(id, docId);
       }
     } else {
-      url = `/api/v1/unstract/${
-        sessionDetails?.orgId
-      }/prompt-studio/prompt-output/?tool_id=${
-        details?.tool_id
-      }&document_manager=${docId}&is_single_pass_extract=${singlePassExtractMode}&profile_manager=${
-        selectedProfile || defaultLlmProfile
-      }`;
+      const orgId = sessionDetails?.orgId;
+      const toolId = details?.tool_id;
+      const profileManager = selectedProfile || defaultLlmProfile;
+      url = `/api/v1/unstract/${orgId}/prompt-studio/prompt-output/?tool_id=${toolId}&document_manager=${docId}&is_single_pass_extract=${singlePassExtractMode}&profile_manager=${profileManager}`;
```

Comment:
NIT: Let's create a base url path if this is a repeating constant as requirement. @tahierhussain

Reply:
@athul-rs As discussed in this [comment](https://github.com/Zipstack/unstract/pull/790#discussion_r1799496594), I'll address this in a separate PR.